ci-operator/step-registry/upi/libvirt/install/upi-libvirt-install-chain.yaml

```diff
@@ -1,8 +1,8 @@
 chain:
   as: upi-libvirt-install
   steps:
-  - ref: ipi-install-rbac
-  - ref: upi-libvirt-install-install
+  - ref: upi-install-libvirt-network
```

Comment:
This file is another case of editing Dominik's workflow. Are we replacing Dominik's network setup with our own? If so, let's pull this out to it's own PR so we don't confuse it with our own UPI specific updates.